package.json

```diff
@@ -185,7 +163,7 @@
       "output": []
     },
     "test:command-reference": {
-      "command": "\"./bin/dev\" commandreference:generate --erroronwarnings",
+      "command": "ts-node \"./bin/dev.js\" commandreference:generate --erroronwarnings",
```

Comment:
oh, is `ts-node` required for this to work on windows? I was getting a failure on windows on my plugin when moving it to esm:
https://github.com/cristiand391/sf-plugin-api/actions/runs/6714528333/job/18248007631?pr=22

Reply:
Yep, that's correct